.github/workflows/config/labeler.yml

```diff
@@ -3,7 +3,7 @@ base_package:
 changelog/no-changelog:
 - requirements-agent-release.txt
 - datadog_checks_dev/datadog_checks/dev/__about__.py
-- all:^(?![^/]+/datadog_checks/.*|[^/]+/pyproject\.toml).*
+- all:['*/!(datadog_checks)/**', '*/!(pyproject.toml)', '*/tests/**']
```

Comment:
Need to also match files outside /datadog_checks/

Reply:
Would this start adding all labels to README changes again? This PR didn't get a `documentation `and `no-changelog` label: https://github.com/DataDog/integrations-core/pull/12889

Reply:
@sarah-witt ooh thanks for raising that! I'll add that back in and test the changes some more